engine/runtime/src/test/java/org/enso/interpreter/test/MetaObjectTest.java

```diff
@@ -1,5 +1,7 @@
 package org.enso.interpreter.test;
 
+import static org.hamcrest.CoreMatchers.*;
```

Comment:
Please avoid `*` wildcard imports.

Reply:
Sorry. Fixed in 44c834291dcb71c5798dc02c060bd52604aadf74. Apparently, our `javafmt` command does not replace star imports, I will have to be more careful about this in the future.